### PR TITLE
React vx bars

### PR DESCRIPTION
**Pull request by srallen**
Package: lib-classifier

Describe your changes:
This is a branch that is a comparison to #1040 but using vx instead of entirely our own React SVG components + d3. It's very similar because I did borrow conceptually from vx, but I very quickly added the bottom axis (took roughly ~10 mins) and swapped out components for vx components. 

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?
- [ ] Is the changelog updated?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?

### CODE REVIEW

File: packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/BarChartViewer/BarChartViewer.spec.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { shallow } from 'enzyme'
+import React from 'react'
+import mockData from './mockData'
```
Comment (rogerhutchings) [inline packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/BarChartViewer/BarChartViewer.spec.js]: https://vx-demo.now.sh/static/docs/vx-mock-data.html might be handy here

**Comment by rogerhutchings** [comment]
This works for me in general, I'll be interested to see how it works out with the LCV.

Minor quibble: the bottom axis isn't quite aligned inside the container.

Screenshot 2019-08-05 15 57 37

File: packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/BarChartViewer/BarChartViewer.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import { withTheme } from 'styled-components'
+import { Bar } from '@vx/shape'
+import { Group } from '@vx/group'
+import { AxisBottom } from '@vx/axis'
+import { scaleBand, scaleLinear } from '@vx/scale'
+import Chart from '../SVGComponents/Chart'
+import Background from '../SVGComponents/Background'
+
+const BarChartViewer = React.forwardRef(function BarChartViewer({ backgroundFill, data, height, theme, width }, ref) {
+  const yMax = height * 0.95
```
Comment (rogerhutchings) [inline packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/BarChartViewer/BarChartViewer.js]: I'd suggest keeping constant values like this in a config file for the component. It's not too much of an issue here, but I can imagine complex charts getting a bit hairy with where these are defined

**Comment by rogerhutchings** [comment]
@srallen What's the plan for this PR, is this still a WIP experiment, shall we get it in a state for merge or shall we trash it?

**Comment by srallen** [comment]
The plan is to get it into a state to merge. I'm currently writing tests for #1074 though. I'll circle back to this after that.

**Comment by srallen** [comment]
@rogerhutchings this is ready for review. I have some additional enhancements to do with this, but don't want to make this any bigger. If requested, I can pull out the new SVG components, `Chart`, and `Background`, into their own PRs to make this smaller.

File: packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/BarChartViewer/BarChartViewer.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+        {data.map((datum, index) => {
+          const { color, label, value } = datum
+          const fill = color || colors.brand
+          const key = `bar-${label}`
+          const barHeight = yMax - yScale(value)
+          const barWidth = xScale.bandwidth()
+          const x = xScale(label)
+          const y = yMax - barHeight
+          return (
+            <Bar
```
Comment (eatyourgreens) [inline packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/BarChartViewer/BarChartViewer.js]: Should these bars have alt text, like the bars in the Daily Classifications Chart?

Comment (srallen) [reply packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/BarChartViewer/BarChartViewer.js]: Yes, that's one of the enhancements I'd like to do after this initial PR

**Pull request merged**